FireboltPrivacy/FireboltPrivacyImplementation.cpp

```diff
@@ -0,0 +1,138 @@
+/*
+ * If not stated otherwise in this file or this component's LICENSE file the
+ * following copyright and licenses apply:
+ *
+ * Copyright 2020 Metrological
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "Module.h"
+#include <interfaces/IConfiguration.h>
+#include <interfaces/IFireboltPrivacy.h>
+
+namespace WPEFramework {
+
+namespace Plugin
+{
+    class FireboltPrivacyImplementation : public Exchange::IFireboltPrivacy,
+                                         public Exchange::IConfiguration {
+
+    private:
+        class Config : public Core::JSON::Container {
+        public:
+            Config(const Config&) = delete;
+            Config& operator=(const Config&) = delete;
+
+            Config()
+                : Core::JSON::Container()
+                , devicemanifest("")
```

Comment:
think "" is not needed, it is empty by default,  now it adds the overhead of creating a std::string first